torch/distributed/pipeline/sync/_balance/profile.py

```diff
@@ -108,7 +116,7 @@ def profile_sizes(
         param_size = sum(p.storage().nbytes() for p in layer.parameters())
 
         # Combine size of parameters and activations with normalize scales.
-        size = latent_size * latent_scale + param_size * param_scale
+        size = latent_size * chunks + param_size * param_scale
```

Comment:
Same here with this `size` calculation -- just want to make sure this is intended

Reply:
As mentioned above, it was my mistake. restored to original Thank you very much.